projects/kit/src/lib/time/utils/parse-time-string.ts

```diff
@@ -0,0 +1,15 @@
+import {MaskitoTimeSegments} from '../types';
+
+/**
+ * @param timeString can be with/without fixed characters
+ */
+export function parseTimeString(timeString: string): MaskitoTimeSegments {
```

Comment:
This function looks like it works with hours down to milliseconds with optional precision, but what if we want to go the other way around and store seconds + milliseconds and optionally add minutes and hours?